lib/constants.js

```diff
@@ -7,6 +7,7 @@ module.exports = {
   AGENT_SETTING: {maxSockets: 20, freeSocketTimeout: 60000}
 , BASE_BACKOFF_MILLIS: 3000
 , MAX_BACKOFF_MILLIS: 30000
+, MAX_ATTEMPS: -1
```

Comment:
This is misspelled, but only here

Reply:
I feel like there's a test somewhere that checks default values. Seems like this could have been caught.

Reply:
oof, fixed.  I can add a unit test but I didn't see anything comparable for other constant definitions so it seemed like overkill.  Happy to do it if you think it's worthwhile.

Reply:
No worries. You shouldn't have to add anything new. This test should take care of it. It's easy to miss because it uses a match which doesm't catch newly-added properties. You should be able to test your symbol here for a -1 value.

https://github.com/logdna/logger-node/blob/fb0639ca88374d584f0c1d5f3003781e997533c9/test/logger-instantiation.js#L34

Reply:
Thanks, I added maxAttempts there.  Sorry for the litany of silly oversights here.